src/ImageSharp/Colors/PackedPixel/IPixel.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ImageSharp
+{
+    using System;
+    using System.Numerics;
+
+    public interface IPixel
```

Comment:
What happens to `PackedValue` ? We need this to be enforced by an interface so that we can provide signature compatibility to MonoGame types. That's why `IPackedVector<T>` still existed despite us not using it.